firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -119,21 +107,47 @@ class CertificatesViewController: UIViewController, Themeable, UITableViewDelega
         ])
     }
 
-    private func setupCertificatesHeaderView() {
-        // TODO: FXIOS-9834 Add tab indicator for table view tabs and hide/unhide it on selection
+    private func setupCertificatesTableView() {
+        certificatesTableView.delegate = self
+        certificatesTableView.dataSource = self
+        view.addSubview(certificatesTableView)
+        NSLayoutConstraint.activate([
+            certificatesTableView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor),
+            certificatesTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            certificatesTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            certificatesTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    // MARK: - UITableViewDataSource
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard !viewModel.certificates.isEmpty else { return 0 }
+        return 4
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerStackView: UIStackView = .build { stack in
+            stack.axis = .horizontal
+            stack.distribution = .fillEqually
+            stack.spacing = UX.headerStackViewSpacing
+        }
         for (index, certificate) in viewModel.certificates.enumerated() {
             let certificateValues = viewModel.getCertificateValues(from: "\(certificate.subject)")
             if !certificateValues.isEmpty, let commonName = certificateValues[CertificateKeys.commonName] {
-                let button: UIButton = .build { [weak self] button in
-                    button.setTitle(commonName, for: .normal)
-                    button.setTitleColor(self?.currentTheme().colors.textPrimary, for: .normal)
-                    button.configuration?.titleLineBreakMode = .byWordWrapping
-                    button.titleLabel?.numberOfLines = 2
-                    button.titleLabel?.textAlignment = .center
-                    button.tag = index
-                    button.addTarget(self, action: #selector(self?.certificateButtonTapped(_:)), for: .touchUpInside)
+                let item: CertificatesHeaderItem = .build { [weak self] item in
+                    guard let self else { return }
+                    item.configure(theme: self.currentTheme(),
+                                   title: commonName,
+                                   isSelected: viewModel.selectedCertificateIndex == index)
```

Comment:
```suggestion
                let item: CertificatesHeaderItem = .build()
                item.configure(theme: self.currentTheme(),
                               title: commonName,
                               isSelected: viewModel.selectedCertificateIndex == index)
```

I suggest this change since it's a little clearer to read. No actual UI logic is needed in `build`.

It would make sense to do `configure` inside the `build` function _IF_ we were unable to follow the variable declaration with some additional logic (e.g. as with a class initializer). But here we can easily do that!